guide/glossary/address.md

```diff
@@ -8,38 +8,78 @@ permalink: /guide/glossary/address/
+### SegWit address - [P2WPKH](https://en.bitcoin.it/wiki/Bech32)
 
-##### Address compatibility
-Since "legacy" addresses are still in use[^1], and some older applications have yet to upgrade. Script addresses can be used to resolve incompatibility issues that can arise when the sender's application does not recognize the SegWit address provided by the receiver.
+A pay-to-witness-public-key-hash (P2WPKH), also known as native SegWit or Bech32 address, is a modern more efficient address format. SegWit addresses are opt-in so not every application supports them but the majority do, and should, today. SegWit adoption can be tracked on [here](https://en.bitcoin.it/wiki/Bech32_adoption).
+ 
+Benefits of SegWit addresses include more resistance to input errors (as they are case insensitive and use error-correcting codes), and lower transaction fees. The fee saving will depend on the type of transaction but for a common transfer of funds it can be 30-40%.
+ 
+SegWit addresses start with `bc1q` and are case insensitive.
+ 
+Example: `bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq`
+
+### Taproot address - [P2TR](https://en.bitcoin.it/wiki/Bech32)
+
+A pay-to-taproot (P2TR), also known as a Taproot or Bech32m address, is the most recent and advanced Bitcoin address format. Taproot introduces more advanced security, privacy, flexibility and scaling to Bitcoin. Like SegWit, Taproot addresses are opt-in and they are not currently widely supported. Taproot adoption can be tracked [here](https://en.bitcoin.it/wiki/Bech32_adoption).
+ 
+Benefits of Taproot include the ability to use Schnorr Signatures offering better security, lower fees, and more flexible multi-signature transactions. Multi-sig addresses using P2TR look the same as single-sig addresses giving users of multisig increased privacy. Taproot also enables more advanced scripting which enables more complex smart contracts to be built on Bitcoin.
+ 
+Taproot addresses start with `bc1p` and are case insensitive.
+ 
+Example: `bc1pmzfrwwndsqmk5yh69yjr5lfgfg4ev8c0tsc06e`
+
+### Change address
+
+When the output of a transaction is used as the input of another transaction, it must be spent in its entirety. Sometimes the value of the output is higher than what the user wishes to pay. In this case, the bitcoin client generates a new Bitcoin address, and sends the difference back to this address. This is known as *change address*. The change address format used will usually be the default format used by the wallet application. 
+
+## Address compatibility
+
+Since "legacy" addresses are still in use, and some older applications have yet to upgrade. Script addresses can be used to resolve incompatibility issues that can arise when the sender's application does not recognize the SegWit address provided by the receiver.
```

Comment:
```suggestion
Since "legacy" addresses are still in use, some older applications have yet to upgrade. Script addresses can be used to resolve incompatibility issues that can arise when the sender's application does not recognize the SegWit address provided by the receiver.
```